test/output/openapi-to-asyncapi/no-channel-parameter.yml

```diff
@@ -0,0 +1,42 @@
+asyncapi: 3.0.0
+info: 
+  title: Sample Pet Store App
+  version: 1.0.1
+  description: This is a sample server for a pet store.
+  termsOfService: 'http://example.com/terms/'
+  contact:
+    name: API Support
+    url: 'http://www.example.com/support'
+    email: support@example.com
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+  tags:
+    name: pet
+    description: Pets operations
+  externalDocs:
+    description: Find more info here
+    url: 'https://example.com'
+
+servers:
+  - url: 'https://{username}.gigantic-server.com:{port}/{basePath}'
```

Comment:
URL is split up in AsyncAPI v3: https://www.asyncapi.com/docs/reference/specification/v3.0.0#serverObject

Reply:
it was done only, i have written the testcase wrongly

Reply:
it was done only, i had forgotten to update the test case